fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientStatus.java

```diff
@@ -63,6 +63,33 @@ public static ClientStatus fromInt(final Integer statusValue) {
         return enumeration;
     }
 
+    public static ClientStatus fromString(final String clientString) {
```

Comment:
Hmmm... how does this actually work now? Here we are comparing to the strings as defined in the enum (e.g. clientStatusType.active) but the test is passing in just "active". So it should not match / return anything. What did I miss here?